### DISCUSSION TITLE
Where is the output panel? Language server crashes, but I can't find the error.

### DISCUSSION THREAD

**Discussion by rpbaptist** [LazyVim/LazyVim#1152] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/7916262/94dc828f-b972-4c33-9992-8da8b4d1fe25)

The message says "See output panel". I don't know where the output panel is.

**Comment by folke** [comment, marked as answer]
There is not real output panel like in vscode. You can check `:LspLog` though

**Reply by rpbaptist** [reply]
Thanks.
